src/ExpressiveInstaller/Resources/config/container-auryn.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+use Auryn\Injector;
```

Comment:
`Auryn` is explicitly used in the file, so it should also be explicity defined as dependency, IMO